numba/cuda/cudadecl.py

```diff
@@ -205,6 +205,18 @@ class Cuda_match_all_sync(ConcreteTemplate):
     ]
 
 
+@register
+class Cuda_activemask(ConcreteTemplate):
+    key = cuda.activemask
+    cases = [signature(types.int32)]
```

Comment:
should the return type be unsigned instead? Consider the situation where the mask is involved in bitshift operation. The current signed int32 type will use arithmetic shift on `>>`. It seems logical shift should be preferred for masks.

Reply:
I think it should be, yeah - this is a typo, I actually intended for it to be unsigned, so I'm glad of this comment!